lib/credit_card_validations/version.rb

```diff
@@ -1,3 +1,3 @@
 module CreditCardValidations
-  VERSION = "2.0.3"
+  VERSION = "3.0.0"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.